advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,461 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "v1",
+		"title": "Microsoft.Azure.Engineering.Gateway"
```

Comment:
Title should be something like XYZManagementClient for management plan as it's used in SDK generation.